express/blocks/cards/cards.css

```diff
@@ -0,0 +1,180 @@
+main .section:has(.card) {
+  background-color: #F1F3F4;
+  padding-top: 80px;
+}
+
+main .section:has(.card)>div>h2:first-of-type {
+  margin-top: 0;
+}
+
+main .section:has(.card) .content {
+  margin: auto;
+  max-width: 375px;
+  padding: 0;
+}
+
+main .section .cards-container>div,
```

Comment:
cards-container doesn't exist anymore. 
Do we need to figure out another way to set this max-width?

Reply:
I'm wondering if this is a mistake. It looks like it would only select child divs of the `card-containers` under the `section` class. But the `card-container` class was set along side the `section` class in express. So unless there is a `card-container` under that then the class would only work if it was `.section.card-container` like it was for the other `card-container` classes that I changed. 

Is there a case where there could be a sub card container in the card container?